npm/react/src/createMount.ts

```diff
@@ -108,7 +108,7 @@ export const makeMountFn = (
             consoleProps: () => {
               return {
               // @ts-ignore protect the use of jsx functional components use ReactNode
-                props: jsx.props,
+                props: jsx?.props,
```

Comment:
I think React defaults to an empty object for no props?

Reply:
Maybe when creating the component but `jsx` here is just the first argument to `mount`, so passing `cy.mount(null)` would throw. I know it doesn't make sense to do that, but the types allow it and React doesn't throw so I thought I'd be safe here.